src/pageObject.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
 };
 
 class ArticlePage extends PageObject {
```

Comment:
ArticlePage
constructor takes url and slug. Constructor should modify original url by adding slug at the end